src/app/space/settings/collaborators/collaborators.component.html

```diff
@@ -47,21 +47,28 @@ <h3 class="collaborators-header">Collaborators of {{context.space.attributes.nam
   </div>
 </div>
 
-<modal #addCollaborators modalClass="chromeless-modal">
-  <modal-content class="chromeless-modal-content">
-    <add-collaborators-dialog [host]="addCollaborators" [spaceId]="context?.space.id" (onAdded)="addCollaboratorsToParent($event)"></add-collaborators-dialog>
-  </modal-content>
-</modal>
-
+<div class="modal fade" bsModal #modalAdd="bs-modal" tabindex="-1" role="dialog" aria-hidden="true" (onShown)="onShowHandler()">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-body">
+          <add-collaborators-dialog #addCollabDialog [host]="modalAdd" [spaceId]="context?.space.id" (onAdded)="addCollaboratorsToParent($event)"></add-collaborators-dialog>
+      </div>
+    </div>
+  </div>
+</div>
 
-<!-- Delete modal -->
-<modal #removeCollaborator title="Remove collaborator from space">
-  <modal-content *ngIf="userToRemove">
-    <div>You are about to remove collaborator {{userToRemove.attributes.fullName}} from this collaboration space.</div>
-    <div>The selected user will not be able to edit or be assigned work in the {{context.space.attributes.name}}.</div>
-    <div class="remove-footer pull-right">
-      <button class="btn btn-default" (click)="removeCollaborator.close()">Cancel</button>
-      <button class="btn btn-danger" (click)="removeUser()">Remove</button>
+<div class="modal fade" bsModal #modalDelete="bs-modal" tabindex="-1" role="dialog" aria-hidden="true" *ngIf="userToRemove">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-body">
+            <div>You are about to remove collaborator {{userToRemove.attributes.fullName}} from this collaboration space.</div>
+            <div>The selected user will not be able to edit or be assigned work in the {{context.space.attributes.name}}.</div>
+            <div class="remove-footer pull-right">
+              <button class="btn btn-default" (click)="modalDelete.hide()">Cancel</button>
+              <button class="btn btn-danger" (click)="removeUser()">Remove</button>
+            </div>
+        </div>
+      </div>
```

Comment:
When I click "Remove from space" on prod, I get the deletion confirmation dialog with the expected text and buttons here. When I do the same locally with this patch applied it brings up the "Add Collaborator" dialog again with the dropdown select.